packages/webpack/src/plugin.js

```diff
@@ -339,6 +353,37 @@ class LavaMoatPlugin {
                   return
                 }
 
+                // Fixes the issue with assets being emitted to dist without a loader
+                // TODO: refactor to move random hardening of the build somewhere it's easier to track.
+                if (
+                  isAmbientAsset(module) &&
+                  module.resource.includes('node_modules') // FIXME: would be better to use canonicalName lookup and match with root
+                ) {
+                  // add a warning about removing the asset
+                  mainCompilationWarnings.push(
+                    new WebpackError(
+                      `LavaMoatPlugin: the following resource was being silently emitted to the dist and LavaMoat has prevented it: '${module.resource}'. If you want to add this resource, explicitly define a file-loader for it in your webpack configuration.`
```

Comment:
```suggestion
                      `LavaMoatPlugin: the following resource was being silently included in the bundle LavaMoat has prevented it: '${module.resource}'. If you want to add this resource, explicitly define a file-loader for it in your webpack configuration.`
```

Reply:
that's not true. 
It doesn't get included in the bundle, it's a separate file alongside the bundle, in the dist folder, that's created (in webpack terms emitted) during compilation.